File: tests/human/x/dart/sort_stable.dart
void main() {
  var items = [
    {'n': 1, 'v': 'a'},
    {'n': 1, 'v': 'b'},
    {'n': 2, 'v': 'c'}
  ];
  var result = List.from(items)..sort((a, b) => (a['n'] as int).compareTo(b['n'] as int));
  print(result.map((e) => e['v']).toList());
}

File: tests/human/x/ts/sort_stable.ts
const items = [
  { n: 1, v: "a" },
  { n: 1, v: "b" },
  { n: 2, v: "c" }
];
items.sort((a, b) => a.n - b.n);
console.log(items.map(i => i.v).join(' '));
